### ISSUE TITLE
Other states?

### ISSUE THREAD

**Issue by LyleSY** [typpo/ca-property-tax#26]
Would it be possible for this to support other states? Property tax is a disaster nationwide, just not as bad as CA.

**Comment by bendnorman** [comment]
Agreed! Does anyone know of an open-source national parcel dataset? [Landgrid](http://landgrid.com/) seems complete but still costs a lot of money. This project seems like a great start given it has already covered ~10% of the country's population.
